CONTRIBUTING.md

```diff
@@ -271,5 +271,63 @@ flag. Below is a list of the available arguments and their purpose:
 
 ---
 
+## Code Formatting
+
+cuDF uses [Black](https://black.readthedocs.io/en/stable/),
```

Comment:
Should probably break this into it's own section since we'll have a C++ (and Java?) section in the future. 
```suggestion
###Python
cuDF uses [Black](https://black.readthedocs.io/en/stable/),
```